setup.py

```diff
@@ -201,6 +201,9 @@ def run(self):
     cmd_classes['bdist_wheel'] = bdist_wheel
 cmd_classes['build_ext'] = build_ext
 
+with open('requirements.txt') as f:
```

Comment:
You may revert the changes in this file first. The others looks good.

Reply:
It looks like there is already some logic for finding the `README.rst`, so I've copied that pattern to find the `requirements.txt`. I can revert it if it doesn't resolve the path correctly.
https://github.com/microsoft/onnxruntime/blob/5aab2671f8346a1489ac8ebf4b364544c6707062/setup.py#L176-L183